### ISSUE TITLE
Update Videotuturial for Phalcon 4

### ISSUE THREAD

**Issue by petrovsa** [phalcon/docs#2517]
In documentation on Tutorial ->Basic Page is videotutorial for Phalcon 3.x. Need to record and update videotutorial for Phalcon 4. Since some things was changed

**Comment by ruudboon** [comment]
Yes! It's on our todo. Thnx for reporting. We keep this open to track the progress.
